Sprintfinity3902/Maps/DefaultMap.cs

```diff
@@ -21,16 +19,14 @@ public class DefaultMap : IMap
 
         private IEntity goriyaBoomerang;
 
-        public DefaultMap()
-        {
+        public DefaultMap() {
```

Comment:
We also need to talk style at the next meeting. Neither of these is "correct" but we should be consistent. I have noticed visual studio automatically places the open bracket is placed on a newline, so I am leaning towards that as our style guide.